### PR TITLE
additional updates from Mike's review

### PR DESCRIPTION
**Pull request by bc-pi**
see https://mailarchive.ietf.org/arch/msg/oauth/cPD_LWWEPedQR3SiU17Juamtwt8/ and https://mailarchive.ietf.org/arch/msg/oauth/ulgKG6F-_RU6S2TaocZpbqEFgzs/

### CODE REVIEW

File: draft-ietf-oauth-selective-disclosure-jwt.md [hunk 1]
```diff
@@ -174,7 +174,7 @@ Issuer:
 :  An entity that creates SD-JWTs.
 
 Holder:
-:  An entity that received SD-JWTs from the Issuer and has control over them.
+:  An entity that received SD-JWTs from the Issuer and has control over them. The term may refer to the actual user, the supporting hardware and software in their possession, or both.
```
Comment (danielfett) [inline draft-ietf-oauth-selective-disclosure-jwt.md:177]:
```suggestion
:  An entity that received SD-JWTs from the Issuer and has control over them. In the context of this document, the term may refer to the actual user, the supporting hardware and software in their possession, or both.
```

**Review by selfissued — APPROVED** [review]
Thanks for these additional updates.

**Comment by bc-pi** [comment]
oh wow... I apparently didn't confirm the merge so this is still pending. I really thought it was done and published -11. Sigh. Sorry. Will look at doing a -12 soonish.

**Comment by bc-pi** [comment]
see #455

**Pull request merged**